install-cni.sh

```diff
@@ -7,16 +7,16 @@ if [ -w "/host/opt/cni/bin/" ]; then
     echo "Wrote CNI binaries to /host/opt/cni/bin/";
 fi;
 
-TMP_CONF='/flannel.conf.default'
+TMP_CONF='/flannel.conflist.default'
 # If specified, overwrite the network configuration file.
 if [ "${CNI_NETWORK_CONFIG:-}" != "" ]; then
 cat >$TMP_CONF <<EOF
 ${CNI_NETWORK_CONFIG:-}
 EOF
 fi
 
-FILENAME=${CNI_CONF_NAME:-10-flannel.conf}
+FILENAME=${CNI_CONF_NAME:-10-flannel.conflist}
```

Comment:
Would a residual `10-flannel.conf` from a previous installation conflict with `10-flannel.conflist` after upgrading? If supporting upgrades is in scope right now we probably want to remove or rename the old default file.

Reply:
Force the cp to ensure overwrite would be better.
cp -f /opt/cni/bin/* /host/opt/cni/bin/;

And also update Dockerfile
ADD flannel.conf.default /flannel.conf.default